yt_dlp/extractor/bilibili.py

```diff
@@ -128,6 +173,9 @@ def extract_formats(self, play_info):
                 }),
                 **parse_resolution(format_names.get(play_info.get('quality'))),
             })
+        if storyboard_format := self._extract_storyboard(
+                float_or_none(play_info.get('timelength'), scale=1000), aid=aid, bvid=bvid, cid=cid):
+            formats.append(storyboard_format)
```

Comment:
I haven't read this PR carefully enough but it seems this download action is embedded inside `extract_format()`, which was assumed and used as a parser function. This may lead to issue when it comes to legacy-format-only videos, where `extract_format()` needs to be called multiple times. IMO it is better to call `_extract_storyboard()` in `BilibiliIE._real_extract()` instead.
Btw, is this new ` _extract_storyboard` made fatal. If so, I'd prefer to keep it non-fatal to avoid causing video download issue when only the storyboard API changes.